py/server/tests/testbase.py

```diff
@@ -74,3 +82,12 @@ def ugp_lock_shared(self):
             yield
         finally:
             j_shared_lock.unlock()
+
+    def assert_table_equals(self, table_a: Table, table_b: Table):
+        self.assertTrue(table_equals(table_a, table_b))
+
+    def assertEqual(self, obj_a, obj_b):
+        if isinstance(obj_a, Table) and isinstance(obj_b, Table):
+            self.assert_table_equals(obj_a, obj_b)
```

Comment:
This feels a little misleading. Why not just insist that code use `assert_table_equals` when that's what it means?

Reply:
I'm fine being more explicit - I can track down the cases that would fail and change them.